package.json

```diff
@@ -34,6 +34,8 @@
     "examples": "webpack-dev-server --config examples/webpack.config.js --no-info --content-base examples",
     "examples-server": "babel-node examples/server.js",
     "flow": "flow check",
+    "format": "prettier --write \"{src,examples}/**/*.js\"",
+    "format-check": "prettier --list-different \"{src,examples}/**/*.js\"",
```

Comment:
I don't think we necessarily need this one easier since eslint will apply **all** the prettier rules to our JS files.

Typically, you see a prettier dirty check when there's something _not_ controlled by eslint prettified like https://github.com/FormidableLabs/spectacle/blob/master/package.json#L27-L29

```js
    "prettier": "prettier \"**/*.{js,json,css,md}\"",
    "prettier-fix": "npm run prettier -- --write",
    "prettier-check": "npm run prettier -- --list-different",
```

Maybe remove this one too and from CI since we're already checking?